### ISSUE TITLE
Generate blog posts from GitHub Action workflows using this GitHub Action

### ISSUE THREAD

**Issue by eriksjolund** [eriksjolund/fedora-coreos-vm#1]
The [Butane](https://github.com/coreos/butane) configurations provided as input to this GitHub action could also be used as input to some auto-generated documentation (for instance text that could be used for blog posts).

Let's say you would like demonstrate some software (that for instance needs to be installed on multiple computers). You could then use this GitHub Action and provide some Butane configurations to it.
The generated documentation could describe the content of the Butane configurations in plain English.

#### An example translation
This Butane configuration
```
variant: fcos
version: 1.1.0
storage:
  files:
    - path: /some/file/path
      contents:
        inline: Just some text
      mode: 0644
      user:
        id: 1000
      group:
        id: 1000
```

could be translated into this English text

-----------------------------------

_Create the file /some/file/path with the content_
```
Just some text
```
_Adjust the file permissions_

```
[root@laptop ~]# chmod 0644 /some/file/path
```

_Adjust the file ownership_

```
[root@laptop ~]# chown 1000:1000 /some/file/path
```

---------------------------

Another possibility is to collect the stdout from the workflow run.
For instance the SSH commands could be listed together with the output.

To summarize this GitHub issue:
Copy-pasting text from the terminal is probably the most common way to create
a blog post describing some command-line tool (or software).
Let's create a script to automate this. The Butane configurations could
be transformed into plain English and we could also collect the stdout (standard output)
from the workflow runs. The logs from the Systemd services could also be collected 
(i.e. `journalctl -u service-name.service` or `journalctl --user -u service-name.service`)
